pom.xml

```diff
@@ -83,7 +87,7 @@
 		<dependency>
 			<groupId>org.junit.vintage</groupId>
 			<artifactId>junit-vintage-engine</artifactId>
-			<version>5.7.0</version>
```

Comment:
here as well

Reply:
Same as above

Reply:
Please roll all the changes you made on the pom file